Aor writing 4 (#180)
symbiotic-engineering/MDOcean: pubs/applied-ocean-research-model/sections/benchmarking.tex

These errors are deemed acceptable for the purposes of this study, since the goal is to demonstrate the value of simultaneously analyzing multiple disciplines and the ability to quickly evaluate a large number of design options.
More details on the WEC-Sim validation are described in \appendixname~\ref{sec:appendix-dynamic-validation}.

\subsection{Static Validation}

The MEEM hydrodynamic coefficients for the nominal RM3 geometry match WAMIT BEM results closely, as previously shown in Fig.~5 of \cite{mccabe_open-source_2024}, except at frequencies below 0.3~rad/s.
This is expected because the BEM solution assumes infinite depth, which requires a finite added mass at low frequency, whereas MEEM uses finite depth, where the added mass must grow logarithmically toward infinity at zero frequency \cite{mciver_added_1991}.
For the site under consideration, the lowest-frequency sea state containing any energy is 0.4~rad/s, so the discrepancy is not a concern.

> Since that paper \cite{mccabe_open-source_2024}, excitation phase has been added to the MEEM implementation and also matches WAMIT results well.

Author: ```suggestion
After the publication of reference~\cite{mccabe_open-source_2024}, excitation phase was added to the MEEM implementation. It also matches WAMIT results well.
```

The structures, geometry, and economics modules are not computationally expensive and together compose the remaining \results[pctRuntimeOther]~of the total runtime. 

% say that it's 210 sea states, that wecsim is parallelized across X cores, that MEEM is using N=M=K=10
\subsection{Discussion}
\Cref{sec:validation-benchmarking} demonstrates that MDOcean achieves accuracies generally within 10\% of established models and benchmarks both at the individual subsystem level and at the system level. 
Just three scale factors are used to tune results across disciplines and achieve an interdisciplinary model fully consistent with all relevant outputs in the reference model report, providing a valuable benchmark to compare technology improvements while retaining the flexibility to adjust scale factors as needed to represent other scenarios.
The model is deemed quite accurate for inputs near the nominal RM3 design and therefore its ability to capture multidisciplinary interactions in a single software package while running 1-3 orders of magnitude faster than established software is considered a major success.

> Still, the model is not perfect, and although only minor tuning was performed, the model's development centers around the Reference Model 3 and there may be unidentified .

Author: ```suggestion
Still, the model is not perfect, and although only minor tuning was performed, the model's development centers around the Reference Model 3 and there may be unidentified discrepancies for geometries far from the reference design.
```
